package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "cabal-phantom-frontend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-cli serve --config ./webpack.config.js --mode development",
+    "build": "webpack --mode production",
+    "start": "npm run build & serve -s ./build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlp-rwanda/cabal-phantom-frontend.git"
+  },
+  "author": "cabal Team",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atlp-rwanda/cabal-phantom-frontend/issues"
+  },
+  "homepage": "https://github.com/atlp-rwanda/cabal-phantom-frontend#readme",
+  "dependencies": {
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "^4.0.0",
```

Comment:
What's `react-scripts` used for?

Reply:
It contains modules of @babel/core , @babel/preset-env and webpack-dev-server, because when i do uninstall it i will be obliged me to install those modules stated above

Reply:
You have them in your devDependencies. Also, even if you needed it, it cannot be in dependencies because it's only used in development.
![Capture d’écran 2020-11-18 à 12 08 40](https://user-images.githubusercontent.com/5022823/99516460-df1d3a80-2996-11eb-9a9c-0999c2f65b9f.png)